pkg/controller/issueapi/validate_code.go

```diff
@@ -39,6 +39,7 @@ func (c *Controller) BuildVerificationCode(ctx context.Context, request *api.Iss
 		RealmID:           realm.ID,
 		IssuingExternalID: request.ExternalIssuerID,
 		TestType:          strings.ToLower(request.TestType),
+		IssuedAt:          now,
```

Comment:
Wouldn't this just be `CreatedAt` (from the gorm.Model)?

Reply:
Yes. I forgot about the gorm.Model fields. it was late